edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/ModuleUpdateMode.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core
+{
+    public enum ModuleUpdateMode
+    {
+        WaitForAll,
+        NonBlocking
```

Comment:
from appsettings_agent it looks NonBlocking is default value, then you should make it first in enum and will default to NonBlocking

Reply:
Default I believe is explicitly configured in c# code, but yes for implicit defaults it is a good idea. Fixed.